asyncua/client/ua_client.py

```
@@ -38,6 +38,7 @@ def __init__(self, timeout=1, security_policy=ua.SecurityPolicy(), loop=None):
         self._callbackmap: Dict[int, asyncio.Future] = {}
         self._connection = SecureConnection(security_policy)
         self.state = self.INITIALIZED
+        self.disconnecting: bool = False
```

isnt'it cleaner to call that variable `closed`? I cannot remember seing software with a variable called disconnecting, while `closed`is common and does the job: We are closed we ignore any error comming

---

@TilmanK do you fix? so we can merge